docs/architecture/adr-038-state-listening.md

```diff
@@ -127,52 +109,31 @@ func (s *Store) Delete(key []byte) {
 
 // onWrite writes a KVStore operation to all the WriteListeners
 func (s *Store) onWrite(delete bool, key, value []byte) {
-    for _, l := range s.listeners {
-        if err := l.OnWrite(s.parentStoreKey, key, value, delete); err != nil {
-            // log error
-        }
-    }
+    s.listener.OnWrite(s.parentStoreKey, key, value, delete)
```

Comment:
What purpose does wrapping this call `s.listener.OnWrite` in `onWrite` serve?

Reply:
It doesn't. Now calling listener in `Set/Delete`